app/models/cms/group.rb

```diff
@@ -4,7 +4,6 @@ class CMS::Group < ApplicationRecord
   # This is BuyerGroup
   self.table_name = :cms_groups
 
-  belongs_to :tenant
```

Comment:
why?

Reply:
this class doesn't exist and breaks when association is queried. But will go to another PR anyway

Reply:
for a reference, see #3875